libenv/unix_iface.c

```diff
@@ -54,6 +54,8 @@
 
 #define CF_IFREQ 2048           /* Reportedly the largest size that does not segfault 32/64 bit */
 #define CF_IGNORE_INTERFACES "ignore_interfaces.rx"
+#define IPV6_PREFIX "ipv6_"
+#define PREFIXED_IPV6_LEN (32 + 7 + sizeof(IPV6_PREFIX))
```

Comment:
There is a `CF_MAX_IP_LENGTH` somewhere I believe.

Reply:
Good point. It's a bit larger than needed, but it probably doesn't matter much:
```
#define CF_MAX_IP_LEN 64
```